gemrb/plugins/GUIScript/PythonConversions.cpp

```diff
@@ -32,13 +32,13 @@ Color ColorFromPy(PyObject* obj) {
 		Color c;
 		// use _signed_ version of Clamp because PyInt_AsLong may return -1 on error and we would like this to = 0
```

Comment:
this comment indicates the use of signed `char` is intentional, but clearly thats an error.

I'm curious what conditions i was seeing, or if I was just trying to be proactive... Let's remove this comment